build/teamcity-local-roachtest.sh

```diff
@@ -18,10 +18,13 @@ run build/builder.sh make bin/roachprod bin/workload bin/roachtest
 tc_end_block "Compile roachprod/workload/roachtest"
 
 tc_start_block "Run local roachtests"
+# Grab a testing license good for one hour.
+COCKROACH_DEV_LICENSE=$(curl "https://register.cockroachdb.com/api/prodtest")
```

Comment:
Do we want to look at the exit status of curl and/or sanity check the output at all?